go/base/static/css/conversations.less

```diff
@@ -43,14 +46,25 @@
     }
   }
 
+  .info-add-state {
+    z-index: 3;
+    position: absolute;
+    width: 380px;
+    margin-left: -190px;
+    left: 50%;
```

Comment:
\o/ slight modification: using `margin: 0 auto` instead to avoid the diagram jumping a bit when the alert is dismissed.

Dōmo arigatō sensei